context.go

```diff
@@ -374,11 +389,11 @@ func (lc *listContent) metadata() map[string]interface{} {
 func castToStrings(slice []interface{}) ([]string, error) {
 	result := make([]string, len(slice))
 	for i, v := range slice {
-		if s, ok := v.(string); !ok {
+		s, ok := v.(string)
```

Comment:
Not sure why this change? Just something that the linter wanted?

Reply:
Yes, and reads a bit better I think.

Reply:
eh, I like the scoping of the other one, I think (though it doesn't matter in this case).